src/applications/yellow-ribbon/containers/YellowRibbonApp/index.jsx

```diff
@@ -136,8 +139,11 @@ export const YellowRibbonApp = ({ hasFetchedOnce }) => (
           <p className="vads-l-col--7">
             Participating school information is for the current academic year.
             To view schools for the previous academic year,{' '}
-            <a href="" rel="noreferrer noopener">
-              download the historical rates (PDF)
+            <a
+              href="https://www.benefits.va.gov/gibill/yellow_ribbon/yellow_ribbon_info_schools.asp"
+              rel="noreferrer noopener"
+            >
+              view the historical rates
             </a>
             .
```

Comment:
Do we need a unit test to confirm that this `.` exists between the `<a>` and the `<p>`? 🕵

Reply:
Do we need a unit test for that unit test? 😉

Reply:
Ah @Mottie now that is good thinking, because sometimes a unit test itself may contain a bug, so we could add another layer of unit tests that tests the unit tests to confirm that the original unit tests are bug-free

Reply:
You know, if software engineering ever gets boring for you you'll always make a great TROLLer @ncksllvn! lol 😜